include/wx/defs.h

```diff
@@ -2699,7 +2697,7 @@ typedef HIShapeRef WXHRGN;
 
 #endif // __WXMAC__
 
-#if defined(__WXMAC__)
+#if defined(__WXMAC__) || defined(__OBJC__)
```

Comment:
This doesn't make sense, there are tests for `__OBJC__` inside this branch.

Maybe we need to test for `__APPLE__` or `__DARWIN__` here?

Reply:
Your suggestion sounds good and should work. I will try it.

Reply:
Fixed